docs/asciidoc/modules/ROOT/partials/usage/config/apoc.mongodb.get.byObjectId.adoc

```diff
@@ -0,0 +1,11 @@
+The procedure support the following config parameters:
+
+.Config parameters
+[opts=header]
+|===
+| name | type | default | description
+| compatibleValues | boolean | true | converts MongoDB data types into Neo4j data types
+| extractReferences | boolean | false | if true and a field contains an ObjectId it will include the related document instead of the ObjectId
```

Comment:
```suggestion
| extractReferences | boolean | false | if true and a field contains an ObjectId, it will include the related document instead of the ObjectId
```